tests/utils.js

```diff
@@ -28,12 +28,12 @@ export function mockDevtoolPostMessage(window) {
     return stub;
 }
 
-export function createComponent(tagName = 'DIV', data = {}, { id = 'component-id', index = 0, depth = 0 } = {}) {
+export function createComponent(name = 'DIV', data = {}, { id = 'component-id', index = 0, depth = 0 } = {}) {
```

Comment:
Can we lower case these tag names?

Reply:
This is the test utilities to help "component" creation 😄 but yeah, we're calling `toUppercase()` on it straight after.